docs/src/pages/components/portal/portal.md

```diff
@@ -26,7 +26,7 @@ You have to wait for the client-side hydration to see the children.
 
 - 📦 [970 B gzipped](https://bundlephobia.com/package/@mui/base@latest)
 
-As the component does not have any styles, it also comes with the unstyled package.
+As the component does not have any styles, it also comes with the Base package.
```

Comment:
```suggestion
As the component does not have any styles, it also comes with the `@mui/base` package.
```

Should we use the actual package name instead?

Reply:
The `@mui/base` is mentioned one line below in the imports example, so I don't think it's necessary.